units/systemd-tpm2-clear.service.in

```diff
@@ -0,0 +1,25 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Issue TPM Clear Request
+Documentation=man:systemd-tpm2-clear.service(8)
+DefaultDependencies=no
+Conflicts=shutdown.target
+After=tpm2.target
+Before=factory-reset.target shutdown.target
+ConditionSecurity=measured-uki
```

Comment:
Why the condition is necessary?

Reply:
we have this condition in tpm2-setup and the pcrextend services too. We only do full-blown initializations and measurements if people buy into the UKI story so far. It doesn't directly have anything to do with UKIs, but indirectly suggests that people are fine with out PCR and TPM use, if they use UKIs.  Hence I figured I just keep the same conditionalization for resetting the TPM too: only if people buy into UKI stuff we'll also take possession of the tpm enough to reset it when needed.

Reply:
or to say this differently: i was afraid to just by default start measuring arbitrary strings into arbitrary PCRs, because people might use them differently, and if we suddenly interfere that would really suck for them. Hence I was looking for a way how we can determine that the admin/user was fine with our use of PCRs, and figure UKI adoption is a good indication, since it has used PCR 11 like we want to use it for a long time, and if people are fine with that, they should also be fine with our other tpm magic.

Reply:
I see. Could you comment that in the unit? Otherwise, someone may be confused later again.